sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/sources/Generator.java

```diff
@@ -18,25 +18,17 @@
 package org.apache.beam.sdk.nexmark.sources;
 
 import static com.google.common.base.Preconditions.checkNotNull;
+import static org.apache.beam.sdk.nexmark.sources.utils.AuctionGenerator.nextAuction;
+import static org.apache.beam.sdk.nexmark.sources.utils.BidGenerator.nextBid;
+import static org.apache.beam.sdk.nexmark.sources.utils.PersonGenerator.nextPerson;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
 import java.io.Serializable;
-import java.util.Arrays;
 import java.util.Iterator;
-import java.util.List;
 import java.util.Objects;
 import java.util.Random;
-import org.apache.beam.sdk.coders.Coder;
-import org.apache.beam.sdk.coders.CoderException;
-import org.apache.beam.sdk.coders.CustomCoder;
-import org.apache.beam.sdk.coders.VarLongCoder;
-import org.apache.beam.sdk.io.UnboundedSource;
-import org.apache.beam.sdk.nexmark.model.Auction;
+
```

Comment:
<!--new_thread; commit:dd2591661935e2ce1c9472d2dd9c401e3011ec9a; resolved:0-->
Actually a tiny nit is to set up imports to just be alphabetized with no spaces except between static and non-static.